test/sql/chunk_utils.sql

```diff
@@ -347,6 +347,8 @@ BEGIN;
     SELECT * FROM test.show_subtables('drop_chunk_test_date');
 ROLLBACK;
 
+SET timezone TO '+0';
```

Comment:
why are you setting timezone to 0 i'd rather see tests run with a timezone to make sure this is handled properly